addons/crm_claim/crm_claim_data.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
```

Comment:
encoding="utf-8" is default encoding. Reference: http://www.w3schools.com/xml/xml_encoding.asp